taggit/models.py

```diff
@@ -15,7 +17,29 @@ def unidecode(tag):
         return tag
 
 
-class TagBase(models.Model):
+class NaturalKeyManager(models.Manager):
+    def __init__(self, natural_key_fields: List[str], *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.natural_key_fields = natural_key_fields
```

Comment:
oh I think we're not actually using this? Since we call `self.model.natural_key_fields` anyways. Maybe I'm missing something